src/proto/streams/recv.rs

```diff
@@ -385,6 +401,7 @@ impl Recv {
 
         if is_ignoring_frame {
             trace!("recv_data frame being ignored on locally reset {:?} for some time", stream.id);
+            self.release_connection_capacity(sz, &mut None);
```

Comment:
So, just to be extra clear (because, I am very tired and I got pretty confused). Could you add a comment that `self.release_connection_capacity` isn't exactly the opposite of `self.consume_connection_window(sz)?;`, and you still need to perform both fn calls to ensure that a `WINDOW_UPDATE` gets sent out?